channeldb/invoice_test.go

```diff
@@ -1066,16 +1098,16 @@ func TestDuplicateSettleInvoice(t *testing.T) {
 	// If we try to settle the invoice again, then we should get the very
 	// same invoice back, but with an error this time.
 	dbInvoice, err = db.UpdateInvoice(ref, nil, getUpdateInvoice(amt))
-	if err != ErrInvoiceAlreadySettled {
-		t.Fatalf("expected ErrInvoiceAlreadySettled")
-	}
+	require.ErrorIs(t, err, invpkg.ErrInvoiceAlreadySettled)
 
 	if dbInvoice == nil {
 		t.Fatalf("invoice from db is nil after settle!")
 	}
 
 	invoice.SettleDate = dbInvoice.SettleDate
-	require.Equal(t, invoice, dbInvoice, "wrong invoice after second settle")
+	require.Equal(
```

Comment:
[Seems that this is a valid format too](https://github.com/lightningnetwork/lnd/pull/6823#discussion_r1070970997) It would not hurt to make it more restrictive though.